account_credit_control/models/credit_control_policy.py

```diff
@@ -44,6 +53,46 @@ class CreditControlPolicy(models.Model):
         default=True,
     )
 
+    @api.depends("auto_process_lower_levels")
+    def _compute_yield_by_level(self):
+        for rec in self:
+            rec.yield_by_level = not rec.auto_process_lower_levels
+
+    @api.multi
+    def write(self, values):
+        res = super(CreditControlPolicy, self).write(values)
+        if 'auto_process_lower_levels' in values:
+            credit_control_model = self.env['credit.control.line']
+            if values['auto_process_lower_levels']:
```

Comment:
@qgroulard Maybe juste one search would do the job (as the domain is the same):

```suggestion
            # As you don't need fields (just recordset with ids)
            new_self = self.with_context(prefetch_fields=False)
            lines = new_self.search(<with a function to retrieve domain?>)
            if values['auto_process_lower_levels']:
```

Reply:
Not relevant anymore, this part of code has been removed for simplification.